IPython/lib/lexers.py

```diff
@@ -201,19 +239,11 @@ def __init__(self, **options):
         # note we need a __init__ doc, as otherwise it inherits the doc from the super class
         # which will fail the documentation build as it references section of the pygments docs that
         # do not exists when building IPython's docs.
-        self.python3 = get_bool_opt(options, 'python3', False)
-        if self.python3:
-            self.aliases = ['ipython3tb']
-        else:
-            self.aliases = ['ipython2tb', 'ipythontb']
 
-        if self.python3:
-            IPyLexer = IPython3Lexer
-        else:
-            IPyLexer = IPythonLexer
+        DelegatingLexer.__init__(
+            self, IPython3Lexer, IPythonPartialTracebackLexer, **options
+        )
```

Comment:
can we use `super()` directly here now ?

Reply:
Yes, it's also mentioned in the DelegatingLexer documentation.